atlasdb-perf/src/main/java/com/palantir/atlasdb/performance/test/api/PerformanceTest.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2015 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.performance.test.api;
+
+import com.palantir.atlasdb.keyvalue.api.KeyValueService;
+
+public interface PerformanceTest {
+
+    void run(KeyValueService kvs, ValueGenerator gen);
```

Comment:
I'm not sure why we hand the value generator to the test rather than just have the test choose and instantiate it's own value generator or generators?